src/components/layout/organize/SingleTaskLayout.tsx

```diff
@@ -0,0 +1,34 @@
+import { FunctionComponent } from 'react';
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+
+import getTask from 'fetching/tasks/getTask';
+import TabbedLayout from './TabbedLayout';
+
+import TaskStatusText from 'components/organize/tasks/TaskStatusText';
+
+
+const SingleCampaignLayout: FunctionComponent = ({ children }) => {
+    const { taskId, orgId, campId } = useRouter().query;
+    const { data: task } = useQuery(['task', taskId ], getTask(orgId  as string, taskId as string));
+
+    if (!task) return null;
+
+    return (
+        <TabbedLayout
+            baseHref={ `/organize/${orgId}/campaigns/${campId}/tasks/${taskId}` }
```

Comment:
We have a challenge here I think, because as has been mentioned in some discussion at some point before, tasks can be accessed both from a campaign directly and from a calendar. And since the breadcrumbs depend on the URL, we've talked about having multiple URLs point to the same NEXT page.

If we do want to go that route, the `baseHref` here should actually depend on the current route, and be either `/organize/1/campaigns/2/tasks/3` or `/organize/1/campaigns/2/calendar/tasks/3`.

Maybe that's causing the bug with the current tab not being selected?

This entire idea of having the same page on multiple URLs feels a little icky, so I'm not sure if we should even do that, or try to fix the breadcrumbs some other way (or just live with it).

Reply:
I think you're right about the root of the bug and I think that we should use `/organize/1/campaigns/1/calendar/tasks/1` since that's how we are dealing with events as planned :+1: but the same page wouldn't be available at multiple URLs with this solution. Only on the calendar/tasks/# url. Maybe I'm missing something.

Reply:
The event page of course hasn't been implemented yet, so I can see that this is confusing, but if you look at the links pointing to the yet-to-exist event page, there are actually several different variations, depending on where the link is rendered.

Link from page | URL of page | Link points to
-|-|-
Campaigns dashboard|/organize/1/campaigns|/organize/1/campaigns/events/1
Global calendar|/organize/1/campaigns/calendar|/organize/1/campaigns/calendar/events/1
Single-campaign page|/organize/1/campaigns/1|/organize/1/campaigns/1/events/1
Single-campaign calendar|/organize/1/campaigns/1/calendar|/organize/1/campaigns/1/calendar/events/1

The reason for this is to achieve this idea of "path based" breadcrumbs, where the path a user has taken to a page is reflected in the breadcrumbs (which in turn are based on the URL). But I'm starting to think that's a moot goal to pursue. I've created a separate issue for this over at #309 which contains more info and where we can continue to discuss and decide.